### DISCUSSION TITLE
Review dla Frontendu

### DISCUSSION THREAD

**Discussion by mattyl006** [emkarcinos/WMIAdventure#388] [Codding]
Jak coś to @Michael-Czekanski i @emkarcinos róbcie też review do tasków Frontendowych, a jak czegoś nie czaicie dlaczego tak zrobiłem to pytajcie w review i dyskusje jak jest propozycja aby coś inaczej zrobić i testujcie zawsze koniecznie czy coś działa tak jak miało działać.

**Comment by itchystone** [comment]
Pewnie, postaram się każdy pull request przeglądać

**Reply by mattyl006** [reply]
Właśnie, i jak chcecie przetestować to checkout na tego brancha i **npm start**, jak pierwszy raz to przed tym **npm install**

**Reply by emkarcinos** [reply]
Ja korzystam z dockera do odpalania stronki u siebie, bo to to samo środowisko co będzie na produkcji.
